webapp/channels/src/components/search/search.tsx

```diff
@@ -285,7 +285,7 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
             return;
         }
 
-        const {error} = await actions.showSearchResults(Boolean(props.isMentionSearch));
+        const {error} = await actions.showSearchResults(Boolean(props.isMentionSearch)) as any;
```

Comment:
What's the `any` here for? Not `ActionResult`?

Reply:
It's because the code for `showSearchResults` got messed up at some point to no longer return an `ActionResult`, and we never noticed because of the old types for `ActionFunc`. `showSearchResults` returns the result of `performSearch` which resolves to `[ActionResult, ActionResult]`, `error` is never actually defined